lib/internal/errors.js

```diff
@@ -309,7 +309,7 @@ function invalidArgType(name, expected, actual) {
 
   // determiner: 'must be' or 'must not be'
   let determiner;
-  if (expected.includes('not ')) {
+  if (typeof expected === 'string' && expected.startsWith('not ')) {
```

Comment:
This should always be a string. A failure would be totally fine out of my perspective as we should have tests for this anyway.
But I think it is not really important for now.